assets/section-main-product.css

```diff
@@ -770,8 +770,8 @@ a.product__text {
 }
 
```

Comment:
Match default settings to Dawn’s current default, currently the toggle is on and the mob thumbs are set to show thumbs. 
VH toggle: Off, 
Desk width: Large, 
Desk layout: Stacked 
Mob layout: Hide thumbs

Reply:
The defaults should be that way. The demo shop was probably just saved with different settings. I've reverted them for now but they could be resaved again at some point.